### ISSUE TITLE
Creating place

### ISSUE THREAD

**Issue by jc222fi** [mw222rs/1DV450_mw222rs#8]
It's possible to create the same place several times. Can't find any specifications in the requirements on this issue so this is merely a suggestion, but in my opinion it shouldn't be possible to add a place that already exists.

**Comment by fyrkant** [comment]
Good idea, I should put in a check for existing places with the same values before I save the new one!
